src/meshapi/models/node.py

```diff
@@ -1,12 +1,19 @@
-from typing import Any
+from typing import TYPE_CHECKING, Any
 
 from django.core.validators import MaxValueValidator
 from django.db import models
 
 from meshapi.util.network_number import NETWORK_NUMBER_MAX, get_next_available_network_number
 
+if TYPE_CHECKING:
+    # Gate the import to avoid cycles
+    from meshapi.models.building import Building
+
 
 class Node(models.Model):
+    # This should be added automatically by django-stubs, but for some reason it's not :(
+    buildings: models.QuerySet["Building"]
+
```

Comment:
This is an attribute annotation, to tell `mypy` that `node` objects have a `buildings` attribute. This is supposed to be automatically added by the reverse relation from the `Building.nodes` relationship, but for some reason `django-stubs` isn't adding it for us. It seemed to be working for the other reverse relationships though, so IDK.

Maybe it's because of the fact that we have multiple Building -> Node relationships due to the `primary_node` column